README.md

```diff
@@ -8,48 +8,18 @@ ERDOS is a platform for building self-driving cars and controlling robots.
 
 The easiest way to get ERDOS running is to use our Docker image:
 
+```console
+docker pull erdosproject/erdos
 ```
-$ git clone https://github.com/erdos-project/erdos
-$ docker build -t erdos .
-$ docker run -it --rm -e DISPLAY=$DISPLAY -v /tmp/.X11-unix:/tmp/.X11-unix erdos
-```
-
-The current Docker image installs Gazebo and allows you to run our implementation
-of the-feeling-of-success pipeline. To run it, just do
-
-```
-$ cd ~/ros_ws
-$ ./intera.sh sim
-$ roslaunch sawyer_gazebo sawyer_world.launch &
-$ cd ~/ros_ws/src/ERDOS/examples/the-feeling-of-success
-$ python run_experiments.py
-```
-
-To install the Carla pipeline, clone the repository and build the Docker image:
-
-```
-$ git clone https://github.com/erdos-project/erdos
-$ git submodule update --init
-$ docker build -t erdos-carla
-$ docker run -p 2000-2002:2000-2002 -it --runtime=nvidia -e NVIDIA_VISIBLE_DEVICES=0 --shm-size="14g" erdos /bin/bash
-```
-
-To run the pipeline, do the following:
-
-```
-$ cd ~/workspace/CARLA_0_8_4/ && ./CarlaUE4.sh -windowed -ResX=8 -ResY=8 -carla-server
-$ cd ~/workspace/ERDOS/examples/benchmarks/pylot && python pylot.py --replay=True
-```
-
 
-# Local instalation
+# Local installation
```

Comment:
I think we should now be able to run on OS X when using Ray as back-end. Can you please update the readme, and ensure that the there's two installation options: with ROS and without ROS?